src/gasman.h

```diff
@@ -1003,6 +1003,12 @@ extern  void            InitCollectFuncBags (
             TNumCollectFuncBags after_func );
 #endif
 
+// ExtraMarkFuncBags, if not NULL, is called during garbage collection
+// This is used for integrating GAP (possibly linked as a shared library) with
+// other code bases which use their own form of garbage collection. For
+// example, with Python (for SageMath) or Julia.
```

Comment:
Not really for Julia, as we are using the Julia GC there.

Reply:
It'd be somewhat helpful if I knew which bits of garbage collection I should care about; at the moment I run with "if it works and tests pass in the master branch, I fire and forget", should I start worrying about boehm and julia gcs?

And if so, I need ways to test them.

Reply:
I don't see how my remarks imply that you should/need to worry about Julia's GC and/or Boehm GC... All I am saying is that for the GAP-Julia interface being created as part of the OSCAR project, `SetExtraMarkFuncBags` won't be used, because there we don't actually use GASMAN; hence "or Julia" should be dropped from that comment.

Reply:
(To clarify: I don't mind if it stays in, or is removed on a later date; feel free to merge right away; I am merely pointing out that it is not based on an actual application that I am aware of).